imagebuildah/build.go

```diff
@@ -222,6 +222,8 @@ type Executor struct {
 	containerIDs                   []string // Stores the IDs of the successful intermediate containers used during layer build
 }
 
+var imageMap map[string]string
```

Comment:
Does this need to be global, rather than a member of the `Executor`?

Reply:
It's not in the executor, the curly at line 223 finishes up the executor.  However to make it stand out more, I'll move the declaration up to line 45ish, just after the const definition and before the Mount  definition.

Reply:
Sorry, I was being unclear:  why isn't it part of `Executor`?  If we had two builds going in a single process, things could get confused.

Reply:
To be honest, hadn't considered the multi build from one process scenario.   Will try moving it to the executor.